chroma-manager/chroma_core/migrations/0023_auto__add_unconfigurersyslogjob__add_getcorosyncstatejob__add_pacemake.py

```diff
@@ -4,6 +4,14 @@
 from south.v2 import SchemaMigration
 from django.db import models
 
+def create_table_dict (model_name, state, host_id):
+    return { k: v for k, v in (
+        ('state_modified_at', datetime.datetime.now()),
+        ('state', state),
+        ('immutable_state', 'f'),
+        ('host_id', host_id),
+        ('corosync_reported_up', 'False' if model_name == 'corosyncconfiguration' else None)
```

Comment:
Everything else in this is static. 

Why not just treat this as a constructor and return a dict?

Reply:
I'm not sure I understand the benefit. I'm not opposed to it, can we discuss?